crispycuts/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "crispycuts",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.11.4",
+    "@testing-library/react": "^11.1.0",
+    "@testing-library/user-event": "^12.1.10",
```

Comment:
Run `yarn remove @testing-library/jest-dom` , `yarn remove @testing-library/react` and `yarn remove @testing-library/user-event`. Then `yarn add -d @testing-library/react`